test/testing/test_pytest_plugin.py

```diff
@@ -136,6 +136,113 @@ def test_container_factory(container_factory, rabbit_config):
     assert get_rabbit_connections(vhost, rabbit_manager) == []
 
 
+def test_container_factory_with_custom_container_cls(testdir):
+
+    testdir.makeconftest(
```

Comment:
no, i mean instead of `makeconftest` with a fake module, you could `makepyfile` with an actual module